vulnerabilities/models.py

```diff
@@ -275,17 +275,27 @@ class Weakness(models.Model):
     vulnerabilities = models.ManyToManyField(Vulnerability, related_name="weaknesses")
     db = Database()
 
+    @property
+    def weakness(self):
+        """
+        Return a queryset of Weakness for this vulnerability.
+        """
+        try:
+            weakness = self.db.get(self.cwe_id)
+            return weakness
+        except Exception as e:
```

Comment:
Avoid too broad `Exception`.

Reply:
@tdruez , the exception raised by cwe2 is this:
```
55             break
     57 if not cwe_obj:
---> 58     raise Exception(f"Invalid CWE ID {cwe_id} - {category or ''}")
     60 return Weakness(*cwe_obj)
```
What should be the best solution here?

Reply:
@TG1999  In that case, you're right, we don't have a choice.
It's a bad design on the cwe2 side.

Reply:
> @TG1999 In that case, you're right, we don't have a choice. It's a bad design on the cwe2 side.

@tdruez How can I improve this design ?